.circleci/config.yml

```diff
@@ -10,12 +10,52 @@ jobs:
           wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
           chmod +x ./architect
           ./architect version
+    - run:  ./architect build
+    - persist_to_workspace:
+        root: .
+        paths:
+        - ./chart-operator
+        - ./architect
+
+  e2eTest:
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/certctl
+    steps:
+    - checkout
+    - attach_workspace:
+        at: .
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./e2e-harness
+    - run: ./e2e-harness localkube
+    - run: ./e2e-harness setup --remote=false
+    - run: ./e2e-harness test --test-dir=integration/test/basic
```

Comment:
Please use our usual approach for multiple integration tests right away: https://github.com/giantswarm/chart-operator/blob/master/.circleci/config.yml#L18

It's really annoying to change up the whole circle file when someone adds the second e2e test otherwise.

Reply:
Sure done.